src/components/IvChart.tsx

```diff
@@ -47,6 +47,9 @@ const getChartLabelFill = (
       ? "hsl(var(--stat-decrease))"
       : "hsl(var(--stat-neutral))";
 
+const getStatValueFill = (value: string) =>
+  value === "31" ? "hsl(var(--stat-max))" : "hsl(var(--stat-neutral))";
```

Comment:
I think we could keep 31 as a constant so we can sue it throughout the codebase. Something like,

const MAX_IV = 31;

We could keep that in a general constants file.

Reply:
Agreed, should be fixed now